arch/arm64/configs/bcm2711_rt_defconfig

```diff
@@ -0,0 +1,1735 @@
+CONFIG_LOCALVERSION="v8-rt"
```

Comment:
I think this needs a leading hyphen, otherwise you get `6.12.19v8-rt+`.

Reply:
Good point. I lost the hyphen when I added a script to generate this for testing. Will fix, I need to test on Pi4 before merging this and offering it as an experimental build.